.github/workflows/build_wheels.yml

```diff
@@ -240,7 +240,7 @@ jobs:
       working-directory: apache-beam-source
       env:
         CIBW_BUILD: ${{ matrix.os_python.python }}
-        CIBW_BEFORE_BUILD: pip install cython
+        CIBW_BEFORE_BUILD: pip install cython && pip install --upgrade setuptools
```

Comment:
Any dangers with always upgrading `setuptools`? I wonder if there a reason this wasn't done before?

Reply:
should be safe to upgrade pip and setuptools.